### DISCUSSION TITLE
dehydrated location

### DISCUSSION THREAD

**Discussion by giacomo-git** [indomus/forum#1132] [HUB: Home Assistant]
Ciao
mi sono accorto di aver configurato dehydrated nel path:
/home/homeassistant/**.homeassistant**/dehydrated

E' possibile spostare nel path 
/home/homeassistant/dehydrated
in modo indolore?

Lo chiedo anche perche vorrei migrare da venv a docker, dovrei quindi instanziare la cartella di configurazione di homeassistant che al momento contiene anche dehydrated. 
Seguendo la guida di InDomus dehydrated è configurato fuori dalla cartella di configurazione
Ho il dubbio che possa rappresentare un problema per la migrazione a docker 

Grazie
Giacomo

**Comment by snakuzzo** [comment]
Sì, puoi spostarla fuori.
Ma comunque se hai intenzione di passare a Docker, dehydrated non ti servirà a niente.
Se seguirai le nostre guide ti troverai ad usare un reverse proxy che, oltre ad esporre in https, ti gestirà anche in autonomia richiesta/rinnovo dei certificati...
invece per l'associazione ip-fqdn ti basterà uno script in crontab, un container docker ad hoc oppure una semplice automazione direttamente su HA.
Comunque è tutta roba che trovi già nelle guide
